### ISSUE TITLE
Styling padding Issue

### ISSUE THREAD

**Issue by rmawatson** [wcoder/highlightjs-line-numbers.js#54]
**Describe the bug**
The script appears to set padding:0 in a <style> tag, meaning I can't override this is the css (as in example in the readme)

`<style type="text/css">.hljs-ln{border-collapse:collapse}.hljs-ln td{padding:0}.hljs-ln-n:before{content:attr(data-line-number)}</style>
`
**To Reproduce**

**Expected behavior**
able to style the padding.

**Screenshots**

**Desktop (please complete the following information):**
 - Chrome
 - Latest

**Smartphone (please complete the following information):**
 - Device:Desktop Windows 10 PC

**Additional context**

**Comment by wcoder** [comment]
Hi @rmawatson thank you for feedback!

Currently, this plugin injects default styles in the `<head>` of the page after init.

## workaround 

You can override default styles if put it after plugin `<script>` declaration.

See an example: https://jsbin.com/wupayip/edit?html,output

**Comment by rmawatson** [comment]
right, yes, but your example with padding-right in the readme is wrong. Due to your padding in the style tag above you cannot set padding-right here. Try in your example code.

```
/* for block of numbers */
td.hljs-ln-numbers {
    -webkit-touch-callout: none;
    -webkit-user-select: none;
    -khtml-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;

    text-align: center;
    color: #ccc;
    border-right: 1px solid #CCC;
    vertical-align: top;
    padding-right: 5px;

    /* your custom style here */
}
```

**Comment by littlefinger42** [comment]
This is due to CSS specificity. Learn more about CSS specificity on [MDN here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).

If you attach your `padding-right: 5px;` to a more specific selector than it is set in the base HLJS styling, it will be applied. A more specific selector could be:

`.hljs-ln td.hljs-ln-numbers { padding-right: 5px; }`

**Comment by rmawatson** [comment]
I know whats it due too... I am pointing out that the example in the readme with this project is wrong.
